src/markdown-pages/docs/cli-command-reference/simbank-cli.md

```diff
@@ -0,0 +1,116 @@
+---
+path: "/docs/cli-command-reference/simbank-cli"
+title: "Exploring Galasa SimBank using the CLI"
+---
+Distributed with Galasa, SimBank is a component that simulates a mainframe application. It sits above another component called SimPlatform, which exists to provide options for future growth. As delivered, SimBank implements a sample banking application against which you can configure and run a set of provided tests in preparation for running your own tests against an *actual* mainframe application. You can also practice writing some new tests to run against the SimBank banking application.
+
+By exercising the Galasa framework against SimBank, you can pre-empt a lot (but not all) of the work and learning necessary to eventually hook your own tests up with a genuine mainframe environment. If the provided SimBank tests do not work, then it is unlikely that you will be able to run your own tests on a mainframe application. In summary, SimBank helps you to learn Galasa's basic principles of operation before you need to learn how to connect Galasa to your own mainframe application-under-test.
+
+## Launching SimBank
+SimBank applications and a set of sample SimBank tests are located in the <a href="https://github.com/galasa-dev/simplatform" target="_blank"> Galasa simplatform repository</a> in GitHub. 
+To start exploring the Galasa Simbank application and to run the sample SimBank tests by using the Galasa CLI, you need to complete the following steps:
+
+1. Clone the Galasa `simplatform` repository on your machine by running the following command in the directory on your local machine in which you want to clone the repository files:
+    ```
+    git clone https://github.com/galasa-dev/simplatform.git
+    ```
+1. Run the `./build-locally.sh` script to build the code.
+1. Run the `./run-locally.sh --server` script to start the simbank server inside a local JVM.
+In a few seconds, the Eclipse *Console* window responds with a series of initialization messages, which on Windows looks like:
```

Comment:
Maybe replace "the Eclipse Console window responds with" with "your terminal will display" - also could be worth explicitly mentioning that the commands will be run in a terminal.

Reply:
+1